data/blog/webview2.md

```diff
@@ -0,0 +1,108 @@
+Neither Electron nor WebView2 is managed by Windows Update.
+
+Both Electron and WebView2 inherit Chromium’s multi-process architecture - namely, a single main process that communicates with one-or-more renderer processes.
+These processes are entirely separate from other applications running on the system.
+Whether you run multiple Electron apps or multiple WebView2 apps, each application will contain a whole copy of the following process architectures:
+
+* ElectronJS Process Model:
+
+    ![ElectronJS Process Model Diagram](/images/Electron-Architecture.png)
+* WebView2 Based Application Process Model:
+
+    ![WebView2 Process Model Diagram](/images/WebView2-Architecture.png)
+
+Read more about [WebView2’s process model](https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/process-model#:~:text=WebView2%20uses%20the%20same%20process%20model%20as%20the,other%20utility%20processes%20as%20described%20in%20that%20article.) and [Electron’s process model](https://www.electronjs.org/docs/tutorial/process-model#:~:text=Process%20Model%20Electron%20inherits%20its%20multi-process%20architecture%20from,applied%20in%20the%20minimal%20quick%20start%20app%20.) here.
+
+Electron provides APIs for common desktop application needs such as menus, file system access, notifications, and more.
+WebView2 is a component meant to be integrated into an application framework such as WinForms, WPF, or Win32.
+WebView2 does not provide operating system APIs outside the web standard via JavaScript.
+
+Node.js is integrated into Electron.
+Electron applications may use any Node.js API, module, or node-native-addon from the renderer and main processes.
+A WebView2 application does not assume which language or framework the rest of your application is written in.
+There are C++ and C# templates available, but in each case your JavaScript code must proxy any operating system access through the application-host process.
+
+Electron strives to maintain compatibility with the web API, including APIs developed from the [Fugu Project](https://fugu-tracker.web.app/).
+We have a [snapshot of Electron’s Fugu API compatibility](https://docs.google.com/spreadsheets/d/1APQalp8HCa-lXVOqyul369G-wjM2RcojMujgi67YaoE/edit?usp=sharing).
+WebView2 maintains a similar list of [API differences from Edge](https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/browser-features).
+
+Electron has a configurable security model for web content, from full-access to full-sandbox.
+WebView2 content is always sandboxed.
+Electron has [comprehensive security documentation](https://www.electronjs.org/docs/tutorial/security) on choosing your security model.
+WebView2 also has [security best practices](https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/security).
+
+The Electron source is maintained and available on GitHub.
+Applications can modify can build their own _brands_ of Electron.
+The WebView2 source is not available on GitHub.
+
+Quick Summary:
```

Comment:
i think it also makes sense to add a line about the update model to the summary since it was a discussion in the paragraphs above